api/debug/flags.go

```diff
@@ -32,73 +32,94 @@ import (
 	"github.com/klaytn/klaytn/log/term"
 	colorable "github.com/mattn/go-colorable"
 	"gopkg.in/urfave/cli.v1"
+	"gopkg.in/urfave/cli.v1/altsrc"
 )
 
 var Memsize memsizeui.Handler
 
 var (
 	verbosityFlag = cli.IntFlag{
-		Name:  "verbosity",
-		Usage: "Logging verbosity: 0=silent, 1=error, 2=warn, 3=info, 4=debug, 5=detail",
-		Value: 3,
+		Name:   "verbosity",
+		Usage:  "Logging verbosity: 0=silent, 1=error, 2=warn, 3=info, 4=debug, 5=detail",
+		Value:  3,
+		EnvVar: "KLAYTN_VERBOSITY",
```

Comment:
Do we need to add "KLAYTN" prefix for all env variables?

Is it possible to follow the previous convention we used in kcnd.conf?
https://github.com/klaytn/klaytn/blob/dev/build/packaging/linux/conf/kcnd.conf

Reply:
I think It can cause unintended configuration.
It will be failed to execute the node in some environment where has `export VERBOSITY="some value"` because `EnvVar` has top priority.
So I just added the prefix, KLAYTN.

Reply:
hmm... Okay. I think there are pros and cons to use prefix "KLAYTN"

pros
- no unintended conflict
cons
- brakes the previous convention
- long environment variable

Maybe there are other pros and cons, let's hear others opinion.

Reply:
@kjhman21 I think that the prefix is required because of the same reason @kjeom suggested.

I provide another project like the followings:
* https://www.runatlantis.io/docs/server-configuration.html#environment-variables
* https://docs.datadoghq.com/agent/guide/environment-variables/

To avoid the duplicated environment' names, most projects are using the prefix.

Reply:
@kjeom @odg0318 Okay got it. Please make sure not to break compatibility